documentation/cli.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Documentation
-section: API
+section: CLI
```

Comment:
Will be better to write "Command line interface" as it will be the title on the page.

Reply:
done